mizar/dp/mizar/workflows/vpcs/create.py

```diff
@@ -35,9 +35,15 @@ def requires(self):
 
     def run(self):
         logger.info("Run {task}".format(task=self.__class__.__name__))
-        v = vpcs_opr.get_vpc_stored_obj(self.param.name, self.param.spec)
-        vpcs_opr.allocate_vni(v)
-        vpcs_opr.create_vpc_dividers(v, v.n_dividers)
-        vpcs_opr.set_vpc_provisioned(v)
-        vpcs_opr.store_update(v)
-        self.finalize()
+        if self.param.body["kind"] == "Network":
```

Comment:
Lets remove this if block since mizar will no longer be watching Arktos networks

Reply:
We still need this right? If we remove this block, how can we look up vpc id according to arktos network name?

Reply:
It will be one in the gRPC server.
Since we are no longer watching Arktos_networks in the vpc triggers, this block will never be entered.